pkgs/development/node-packages/node-env.nix

```diff
@@ -363,7 +363,7 @@ let
 
         npm ${forceOfflineFlag} --nodedir=${nodeSources} ${npmFlags} ${stdenv.lib.optionalString production "--production"} rebuild
 
-        if [ "''${dontNpmInstall-}" != "1" ]
+        if [ "$dontNpmInstall" != "1" ]
```

Comment:
and this. There are open PRs against node2nix to fix both...

Reply:
Also done, thanks! Any plans to cache parts of the nix so that adding node packages doesn't reload the whole npm cache? Would you accept contributions on that?

Reply:
How would that work? For patches against node2nix it's better to ask its maintainer.